.github/workflows/test.yaml

```diff
@@ -130,7 +130,7 @@ jobs:
           --request DELETE
           --header 'Accept: application/vnd.github.v3+json'
           --header 'Authorization: Bearer ${{ github.token }}'
-          "$GITHUB_API_URL/repos/$GITHUB_REPOSITORY/actions/caches?key=docker-cache-test-${{
+          "$GITHUB_API_URL/repos/$GITHUB_REPOSITORY/actions/caches?key=docker-cache-test-${{ matrix.os }}-${{
```

Comment:
Nit: recommend following the existing style of putting each context on its own line since this line is fairly long.